docs/releases/2.0.3.txt

```diff
@@ -0,0 +1,13 @@
+==========================
+Django 2.0.3 release notes
+==========================
+
+*Expected March 1, 2018*
+
+Django 2.0.3 fixes several bugs in 2.0.2.
+
+Bugfixes
+========
+
+* Fixed regression in the use of sliced ``QuerySet.order_by().distinct()``
```

Comment:
Fixed regression that caused sliced ``QuerySet.order_by().distinct()`` followed by ``count()`` to crash